api/handlers/jobs.ts

```diff
@@ -257,3 +258,15 @@ export async function snootyBuildComplete(event: APIGatewayEvent): Promise<APIGa
     body: `Snooty build ${jobId} completed`,
   };
 }
+
+/**
+ * Assembles Gatsby Preview URL address for Job post-build
+ * @param payload
+ * @returns string|undefined
+ */
+function getPreviewUrl(payload?: Payload): string | undefined {
+  if (!payload) return;
+  const { repoOwner, branchName, project } = payload;
+  const githubUsernameNoHyphens = repoOwner.split('-').join('').toLowerCase();
+  return `https://preview-mongodb${githubUsernameNoHyphens}.gatsbyjs.io/${project}/${branchName}/index`;
```

Comment:
If we wanted to support pre-production sites for Gatsby Cloud, we should probably have a way to differentiate them by name. Is it possible to do something like check the Autobuilder's current environment, and if it's `stg`, append `stg` after `preview-mongodb${githubUsernameNoHyphens}`?

Reply:
That sounds perfect!

Reply:
@rayangler changed! However, I always get confused about this. Should it encompass '`dotcomstg`' as well?

Reply:
`stg` webhooks should be reserved for GitHub push/content staging jobs while `dotcomstg` should be for test deploys, to sort of emulate the expected `prd` vs. `dotcomprd` behavior in production (but I don't think we enforce nor care too much, especially in pre-production). I suggested only `stg` under the assumption that we'd only ever use this webhook for either `prd` or `stg` data for now